CONTRIBUTING.md

```diff
@@ -1,36 +1,69 @@
 Contributing to the Mesos Cookbook
 ====================================
-The Mesos cookbook uses GitHub to triage, manage, and track issues and pull requests to the cookbook. GitHub has excellent documentation on how to [fork a repository and start contributing](https://help.github.com/articles/fork-a-repo.).
+The Mesos cookbook uses GitHub to triage, manage, and track issues and pull
+requests to the cookbook. GitHub has excellent documentation on how to
+[fork a repository and start contributing](https://help.github.com/articles/fork-a-repo.).
 
-All contributors are welcome to submit patches, but we ask you keep the following guidelines in mind:
+All contributors are welcome to submit patches, but we ask you keep the
+following guidelines in mind:
 
 - [Coding Standards](#coding-standards)
 - [Testing](#testing)
 - [Prerequisites](#prerequisites)
 
-If you're looking to contribute but aren't sure where to start, check out the open issues.
+Please also keep in mind:
+
+- Be patient as not all items will be tested or reviewed immediately by the core
+team.
+- Be receptive and responsive to feedback about your additions or changes. The
+core team and/or other community members may make suggestions or ask questions
+about your change. This is part of the review process, and helps everyone to
+understand what is happening, why it is happening, and potentially optimizes
+your code.
+- Be understanding
+
+If you're looking to contribute but aren't sure where to start, check out the
+open issues.
+
+
+Will Not Merge
+--------------
+This second details Pull Requests that we will **not** merge.
+
+1. New features without accompanying Test Kitchen tests
+1. New features without accompanying usage documentation
```

Comment:
Pro Tip:  If you number every numbered item with 1. markdown takes care of the actual order.  Saves time later if you add or remove items.